docs/how-to/saml-configuration.md

```diff
@@ -0,0 +1,7 @@
+# How to configure SAML
+
+To configure the SAML Integrator integration you'll have to set the following configuration options with the appropriate values for your SAML server by running `juju config [charm_name] [configuration]=[value]`.
```

Comment:
I think the filename should be something according to what is being explained, like:
https://github.com/canonical/indico-operator/blob/main/docs/how-to/configure-saml.md?plain=1

More guidelines extracted from ISD-525 (when the Doc team reviewed our docs):

Tutorials section: rename to tutorial (singular) - [why](https://canonical.com/blog/the-idea-of-a-tutorial)

“what you will learn” > “what you will do”

cross-test tutorials with other team members

How to guides - each title should help form a sentence

“how to contribute” “how to cache content”

In the menu the wording is “how to guides”, change to “How to”

sub-menus contain the action (So “How to” > “Contribute”, “How to” > “Cache content” etc.)

The sub-page title contains the whole title, e.g. “How to contribute”, “How to cache content” etc.)

Capitalization - First letter of each heading only (European style)